infrastructure/data_access/synthetics_repo_localfile.py

```diff
@@ -0,0 +1,30 @@
+import pickle
+
+from domain.model import MeshResults
+from domain.types import TestID
+
+
+class SyntheticsRepoLocalFile:
+    """
+    SyntheticsRepoLocalFile implements domain.Repo protocol.
+    It allows to load/store MeshResults to a file.
+    """
+
+    def __init__(self, mesh_test_results_filename: str) -> None:
```

Comment:
```suggestion
    def __init__(self, mesh_test_results_file_path: str) -> None:
```

This is not file name, but a path. The same with other "filename" occurrences

https://docs.python.org/3/library/functions.html?highlight=open#open
> file is a path-like object giving the pathname (absolute or relative to the current working directory) of the file to be opened or an integer file descriptor of the file to be wrapped.